packages/yoga/src/ActionRequirement/web/ActionRequirement.jsx

```diff
@@ -73,15 +73,18 @@ function ActionRequirement(props) {
   defineCompoundComponents();
   return (
     <StyledActionRequirement {...props}>
-      {illustration && <BoxIllustration>{illustration}</BoxIllustration>}
-
+      {illustration && (
+        <BoxIllustration aria-hidden="true">{illustration}</BoxIllustration>
+      )}
       <Content>
         {titleAsTextDisplay ? (
-          <Text.Display2>{title}</Text.Display2>
+          <Text.Display2 as="h1" aria-label={title}>
+            {title}
+          </Text.Display2>
         ) : (
-          <Title>{title}</Title>
+          <Title aria-label={title}>{title}</Title>
         )}
-        <Text.Body1 mt="small" color="deep">
+        <Text.Body1 mt="small" color="deep" aria-label={description}>
```

Comment:
We don't need to add an `aria-label ` with the same value as children, the value of children will already be detected by assistive technologies.